mergin/client.py

```diff
@@ -784,17 +787,19 @@ def add_user_permissions_to_project(self, project_path, usernames, permission_le
         :param usernames: list of usernames to be granted specified permission level
         :param permission_level: string (reader, writer, owner)
```

Comment:
please add `editor` here as well, and a note that it's only supported on server >= 2024.4